index.js

```diff
@@ -6,7 +6,7 @@ module.exports = function (bot, options) {
   const express = options.express || require('express')
   const app = options.app || express()
   const http = options.http || require('http').createServer(app)
-  const io = options.io || require('socket.io')(http)
+  const io = options.io || require('socket.io').listen(http)
```

Comment:
Since socket.io 3.X.X `.listen` is not needed (https://github.com/imharvol/mineflayer-web-inventory/pull/13)

Reply:
done